src/views/Home/components/Milestones/index.tsx

```diff
@@ -1,9 +1,38 @@
 import * as React from "react"
 import "./index.scss"
+import { MilestoneProps } from "../../../../Props/props"
+import milestoneResults from "../../../../Props/milestones/milestones"
+import milestonesBorderMobile from "../../../../assets/milestones/milestone_header_mobile.svg"
```

Comment:
We can import relative to the `src` folder because of [this](https://github.com/CruzHacks/cruzhacks-2022-website/blob/development/tsconfig.json#L9). I think that will reduce the number of `../`s.